### ISSUE TITLE
[Bug] [UWP] ScrollView issue within RefreshView

### ISSUE THREAD

**Issue by LakshmiNatarajan21** [xamarin/Xamarin.Forms#11326]
### Description
The Entry focus is not updated properly when added inside ScrollView which loaded inside RefreshView. The issue occurs from XF 4.4. (Not sure about the exact workable version). The same working fine in Android and not tested in iOS.

This issue is the same mentioned in the below report,
https://github.com/xamarin/Xamarin.Forms/issues/11106

The same issue occurs if we add ScrollView inside the **RefreshView**.

### Steps to Reproduce

1. Run the sample in UWP.
2. Scroll down.
3. Tap on entry. 
Note: Entry does not get the focus.

### Expected Behavior
Entry should get the focus on clicking.

### Actual Behavior
Entry does not get the focus.

### Basic Information

- Version with issue:
- Last known good version:
- IDE:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:  Yes
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

### Reproduction Link

[IssueSample](https://www.syncfusion.com/downloads/support/directtrac/general/ze/IssueSample2090359060)

### Workaround
NA

**Comment by PureWeen** [comment]
@LakshmiNatarajan21 are you still seeing this issue with the latest XF?

https://www.nuget.org/packages/Xamarin.Forms/

**Comment by LakshmiNatarajan21** [comment]
@PureWeen , Yes, the same issue also occurred in the latest version 4.7.0.1080.

**Comment by bmacombe** [comment]
It's always works for whatever entries are displayed without scrolling, so even if you resize the window, entries that previously worked before they were below the scroll, then don't once they move off-screen and you scroll to them

**Comment by bmacombe** [comment]
I tried to replicate in a standard UWP app, seems to work fine there.
![image](https://user-images.githubusercontent.com/15127788/87184602-ed9f6b00-c2ad-11ea-8e8e-ad6d72fcf637.png)

**Comment by chrisevans9629** [comment]
In case this helps someone else until this gets fixed, this is what we did to get around this issue:

```xaml

<Grid.Resources>
    <ScrollView x:Key="mainUI" />
</Grid.Resources>

<OnPlatform x:TypeArguments="View">
            <On Value="{StaticResource mainUI}" Platform="UWP"/>
            <OnPlatform.Default>
                <RefreshView IsRefreshing="{Binding IsBusy}" Command="{Binding RefreshCommand}" Content="{StaticResource mainUI}"/>
            </OnPlatform.Default>
</OnPlatform>
```
This is removes the RefreshView when running on UWP.

**Comment by javanjoel** [comment]
Thasnk @chrisevans9629 ... It's ashamed that we have to even do these workarounds in Xamarin. Seems like 80% of my time in Xamarin is figuring out workarounds for all these quirks.
